### DISCUSSION TITLE
对战平台

### DISCUSSION THREAD

**Discussion by culaccino0** [ymcui/Chinese-LLaMA-Alpaca-2#20] [General]
对战平台是不是还没集成Chinese-Alpaca-2-7B和Chinese-LLaMA-2-7B，想看看他们和一代效果的比较

**Comment by ymcui** [comment]
已集成。因为对战地址改变（清空cookies和缓存，或者用隐私窗口打开），所以可能会指向旧地址。

**Reply by culaccino0** [reply]
![image](https://github.com/ymcui/Chinese-LLaMA-Alpaca-2/assets/86782918/19ac511f-f2b1-40fe-bbb2-13884d290f3b)
现在直接打不开了，无痕模式也是这个样子，是我的问题吗😂

**Reply by ymcui** [reply]
http://35.202.232.108:7866

**Reply by culaccino0** [reply]
感谢😘

**Reply by Pycelle** [reply]
想看看跟别的模型的对比效果
